.travis.yml

```diff
@@ -1,2 +1,23 @@
 language: d
+dist: trusty
 sudo: false
+
```

Comment:
Actually it is pretty rare to find something that's get's broken only in between nightly releases. Most of the time, once it gets in it stays until the second beta or the next release. That's why I don't think it would be a maintenance problem for us.
There are generally two cases of breakage cause by upstream (dmd/druntime/phobos):
* intentional
* unintentional

If it is intentional, that means that the behavior is most likely here to stay, so we'll need to fix our code sooner or later. There's really no reason to wait until the beta to fix the problem as this will just waste user's time.
If it's unintentional that means the bug needs to be caught as early as possible and reported / fixed upstream, before being officially released and causing public mayhem.

Of course, if using `dmd-nightly` becomes a maintenance problem, we can always make it optional, but I think it is our duty to maintain ecosystem stability and testing bleeding edge is key part of the solution.